app/controllers/items_controller.rb

```diff
@@ -36,6 +36,7 @@ def create # rubocop:disable Metrics/AbcSize, MethodLength, Metrics/PerceivedCom
   def update # rubocop:disable Metrics/AbcSize
     @item.name = params[:item][:name] if params[:item][:name].is_a?(String)
     @item.maori_name = params[:item][:maori_name] if params[:item][:maori_name].is_a?(String)
+    @item.notes = params[:item][:notes] if params[:item][:notes].is_a?(String)
```

Comment:
After discussing with Daniel we're deciding to use permitted params here instead of individual checks for a more standards based approach

Reply:
This is now in :)